server/index.js

```diff
@@ -80,6 +80,13 @@ export default class Server {
       const p = join(__dirname, '..', 'client', ...(params.path || []))
       await serveStatic(req, res, p)
     })
+
+    this.router.get('/_webpack/:number', async (req, res, params) => {
```

Comment:
So, shall we add this route only in the production mode. So we are sure, we don't interfere with the webpack dev server.